django/contrib/sessions/backends/cached_db.py

```diff
@@ -57,10 +57,11 @@ def save(self, must_create=False):
             self._cache.set(self.cache_key, self._session, self.get_expiry_age())
         except UpdateError:
             raise
-        except Exception:
-            # import logging
-            # log the error
-            pass
+        except Exception as e:
+            import logging
+
+            logger = logging.getLogger("django.request")
+            logger.error("Error saving to cache: %s", e)
```

Comment:
Optionally we can do `logger.exception(<message>)` which captures the exception:

https://docs.python.org/3/library/logging.html#logging.Logger.exception

This hasn't been used in Django yet so let's wait to see what Mariusz thinks.